utils/modularize.js

```diff
@@ -129,16 +135,26 @@ function convert( path, ignoreList ) {
 
 	//
 
-	var keys = Object.keys( dependencies )
-		.filter( value => value !== className )
-		.map( value => value === '_Math' ? 'Math as _Math' : value )
-		.map( value => '\n\t' + value )
-		.sort()
-		.toString();
-	var imports = `import {${keys}\n} from "../../../build/three.module.js";`;
+	const threeKeys = [];
+	const moduleDepencies = [];
+	Object.keys( dependencies ).forEach(value => {
+		if (value === className) {
+			return;
+		}
+		const file = files.find(file => file.path.endsWith(`/${value}.js`));
+		if (file) {
+			moduleDepencies.push(file.path);
+			return;
+		}
+		threeKeys.push(value === '_Math' ? 'Math as _Math' : value);
+	});
```

Comment:
Can you explain your reasoning for the changes you've made here?

Reply:
There are dependencies to other modules in the examples folder e.g. `TgaLoader`. This change checks if the detected dependency is one of the other known example-modules. If so, that module is imported. Otherwise it is treated and imported as a "core" `THREE`-module.

Reply:
Can we do this instead?

```js
{ path: 'loaders/FBXLoader.js', dependencies: [ 'curves/NURBSCurve', 'loaders/TGALoader' ], ignoreList: [] },
```

The less magic we have in the script the easier it'll be to maintain.

Reply:
> Can we do this instead?

I get your point. But the detection of dependencies is already part of the script, but they cannot be "resolved" without this change. Besides: we probably don't want to adjust and/or review the `modularize`-script every time an example is updated and changes it's dependencies....